sdk/eventhub/azure-eventhub/tests/unittest/test_connection_string_parser.py

```diff
@@ -0,0 +1,71 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import os
+import pytest
+from azure.eventhub import (
+    EventHubConnectionStringProperties,
+    parse_connection_string,
+)
+
+from devtools_testutils import AzureMgmtTestCase
+
+class EventHubConnectionStringParserTests(AzureMgmtTestCase):
+
+    def test_eh_conn_str_parse_cs(self, **kwargs):
+        conn_str = 'Endpoint=sb://eh-namespace.servicebus.windows.net/;SharedAccessKeyName=test-policy;SharedAccessKey=THISISATESTKEYXXXXXXXXXXXXXXXXXXXXXXXXXXXX='
```

Comment:
please test on EH connection strings - take an example from the portal and "fakify" it

Reply:
this applies to all the tests ^^

Reply:
so this endpoint is one that I "faked" by copying an actual EH endpoint and substituting all info. I changed the namespace and policy names from the SB tests, but I can change it back for consistency?